CMakeLists.txt

```diff
@@ -15,6 +15,7 @@ endif()
 
 set(LIBIIO_VERSION_MAJOR ${PROJECT_VERSION_MAJOR})
 set(LIBIIO_VERSION_MINOR ${PROJECT_VERSION_MINOR})
+set(LIBIIO_VERSION_PATCH 0)
```

Comment:
This should be using `${PROJECT_VERSION_PATCH}` per normal/standard [cmake](https://cmake.org/cmake/help/latest/variable/PROJECT_VERSION_PATCH.html) semantics.